.github/workflows/release.yml

```diff
@@ -18,11 +18,11 @@ jobs:
           - target: x86_64-apple-darwin
             archive: zip
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v3
       - name: Compile and release
-        uses: rust-build/rust-build.action@master
```

Comment:
Patch level is definitely too fine-grained, we don't want to update the action on _every_ release of `rust-build.action`. `@v1` should be fine.